torchdr/tests/test_neighbor_embedding.py

```diff
@@ -38,6 +40,39 @@ def toy_dataset(n=300, dtype="float32"):
     return X.astype(dtype), y
 
 
+@pytest.mark.parametrize("dtype", lst_types)
+@pytest.mark.parametrize("keops", lst_keops)
+def test_array_init(dtype, keops):
+    n = 300
+    X, y = toy_dataset(n, dtype)
+
+    Z_init_np = np.random.randn(n, 2).astype(dtype)
+    Z_init_torch = torch.from_numpy(Z_init_np)
+
+    lst_Z = []
+    for Z_init in [Z_init_np, Z_init_torch]:
+        model = TSNE(
```

Comment:
the init should work on all estimator no? maybe do  a loop? you can decrease the number of iteration ti keep tests fast.